wordpress-mobile/WordPress-iOS
# Plugin Directory empty state

This turned out to be a bit bigger than I hoped, because when I was adding the empty state and refresh button, I realised it's kinda broken in the list view, and when I was already there I refactored the featured plugin code to not require a passed `site`...

Hopefully it ain't too big though.

Testing:
1. Turn on Network Link Conditioner, with "100% loss" preset.
2. Go to plugins.
3. Verify that it shows an empty state  indicating that it's loading plugins.
4. Let it timeout
5. Verify that it shows an error state.
6. Turn off the Network Link Conditioner
7. Tap on the "Try Again" button
8. Verify that the plugins loaded correctly
